packages/client/src/object/errors/handlers/ExecuteActionErrorHandler.ts

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ValueType } from "@osdk/gateway/types";
+import type {
+  ActionEditedPropertiesNotFound,
+  ActionNotFound,
+  ActionParameterObjectNotFound,
+  ActionParameterObjectTypeNotFound,
+  ActionTypeNotFound,
+  ActionValidationFailed,
+  ApplyActionFailed,
+  FunctionEncounteredUserFacingError,
+  FunctionExecutionTimedOut,
+  FunctionInvalidInput,
+  InvalidGroupId,
+  InvalidUserId,
+  MissingParameter,
+  OntologyEditsExceededLimit,
+  ParameterObjectNotFound,
+  ParametersNotFound,
+  PropertiesNotFound,
+  PropertyApiNameNotFound,
+  PropertyBaseTypeNotSupported,
+  UnknownParameter,
+} from "../ApiErrors.js";
+import type { PalantirApiError } from "../Errors.js";
+import { DefaultErrorHandler } from "./DefaultErrorHandler.js";
+
+export class ExecuteActionErrorHandler extends DefaultErrorHandler {
```

Comment:
Do not need all these error handlers, basically just ported over everything from 1.1 for right now